content/docs/manage/operations.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Operations
+enableTableOfContents: true
+isDraft: false
+---
+
+An operation is an action performed on a Neon resource. Operations can be user-initiated or initiated by the Neon Control Plane. User-initiated operations result from actions such as creating a branch or deleting a database. Operations initiated by the Neon Control Plane may include suspending a compute instance after a period of inactivity or checking the availability of a compute instance, for example. You can monitor operations to keep an eye on the overall health of your Neon project or to check the status of specific operations.
+
+Operations include:
+
+- `apply_config`: Applies a new configuration to a Neon object or resource
+- `check_availability`: Checks the availability of an endpoint compute resource
+- `create_timeline`: Creates a project
+- `delete_tenant`: Deletes the storage data
+- `delete_timeline`: Deletes a branch
+- `pageserver_tenant_detach`: Detaches the storage data from the old Pageserver
+- `pageserver_tenant_migrate`: Attaches the storage data to the new Pageserver
```

Comment:
```suggestion
- `pageserver_tenant_migrate`: Attaches the storage data to the Pageserver to perform the migration of the project between Neon Pageserver nodes
```

not sure about the wording here, but perhaps this could work?

Reply:
also include the link to the pageserver doc?